services/admin/src/server.ts

```diff
@@ -293,6 +303,146 @@ export function buildApp({ store }: { store: Store }): Application {
     );
   });
 
+  /* istanbul ignore next */
+  app.get<
+    { electionId: Id; writeInId: Id },
+    Admin.GetWriteInImageResponse,
+    Admin.GetWriteInImageRequest
+  >(
+    '/admin/elections/:electionId/cvr-file/:writeInId/write-in-image',
```

Comment:
Per @jonahkagan I haven't been nesting things like this. Also, it should be `cvrFileId`, right?

```suggestion
    '/admin/cvr-files/:cvrFileId/write-in-image',
```

Reply:
well I'm also using the election id as I need election information, but the other id is for the write in id, I should rename the endpoint though I made it cvr-files when I was feeding through the cvrId and then realized I needed the writeInId (as I need the contestId and optionId)

Reply:
oh I guess I could get the electionId from the writeInId too duh